src/main/java/Krot.java

```diff
@@ -0,0 +1,146 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.util.List;
+
+public class Krot {
+    static String botName = "Krot"; // Static variable for the bot's name
```

Comment:
Consider making this a static final variable and use the proper naming convention for constants. Constant names must be all uppercase using underscore to separate words (aka SCREAMING_SNAKE_CASE).

https://se-education.org/guides/conventions/java/basic.html#naming